### ISSUE TITLE
mergeLonghand breaks border-color

### ISSUE THREAD

**Issue by hudochenkov** [cssnano/cssnano#553]
`mergeLonghand` breaks CSS in `cssnano@4.0.2`.

Input:

```css
.a {
    border: solid transparent;
    border-width: 0 8px 16px;
    border-bottom-color: #eee;
}
```

Output:

```css
.a {
  border: solid transparent;
  border-color: currentColor currentColor #eee;
  border-width: 0 8px 16px;
}
```

`currentColor` is black by default, so instead of transparent, we have black.

**Comment by andyjansson** [comment]
Looking into this.

**Comment by evilebottnawi** [comment]
Fixed in `cssnano@4.0.3`

**Comment by hudochenkov** [comment]
Thank you for a quick fix!
